packages/liquid-html-parser/src/stage-2-ast.ts

```diff
@@ -1103,7 +1109,7 @@ export function cstToAst(
 }
 
 function buildAst(
-  cst: LiquidHtmlCST | LiquidCST | ConcreteAttributeNode[],
+  cst: LiquidHtmlCST | LiquidCST | LiquidDocCST | ConcreteAttributeNode[],
```

Comment:
Should I just be adding this to `LiquidHtmlCST` ? My plan is to allow `LiquidDocCST` to evolve over time. I wanted to keep them separate as much as possible since `LiquidDoc` is kind of its own thing

cc: @charlespwd